app/src/main/kotlin/com/wire/android/ui/calling/ProximitySensorManager.kt

```diff
@@ -73,14 +74,20 @@ class ProximitySensorManager @Inject constructor(
                 coreLogic.globalScope {
                     when (val currentSession = currentSession()) {
                         is CurrentSessionResult.Success -> {
+                            try {
+                                if (wakeLock.isHeld) {
+                                    wakeLock.release()
+                                }
+                            } catch (exception: Exception) {
```

Comment:
so what was the reason of that crash? Can't we handle it somehow else

Reply:
Hey @borichellow , actually this PR wasn't meant to be reviewed yet as this PR was mostly just a test case for auto assigning reviewers.

But based on the implementation code of `.release()` it crashes due to trying to release a wakeLock that doesn't hold anything, thus wrapping it up in a try catch would be enough as it tries to release something it doesn't have a reference/nor holds anything.

Currently it only happens to Xiaomi devices through Android 9 ~ 11, I will be getting one Xiaomi device today later on and will be able to properly do the tests as the crash has not happened in Emualtors/Samsung/Pixel devices I tested.